src/js/stores/MesosStateStore.js

```diff
@@ -264,8 +272,9 @@ class MesosStateStore extends GetSetBaseStore {
     if (framework) {
       const activeTasks = framework.tasks || [];
       const completedTasks = framework.completed_tasks || [];
+      const unreachableTasks = framework.unreachable_tasks || [];
 
-      return activeTasks.concat(completedTasks);
+      return activeTasks.concat(completedTasks).concat(unreachableTasks);
```

Comment:
why not re-use the first `concat(completed_tasks, unreachable_tasks)` as one operation?